app/frontend/controllers/app_controller.js

```diff
@@ -480,20 +524,22 @@ class AppController {
       return false;
     });
   }
-
-  toggleFullScreen() {
+  
+  getPlatform() {
     var platform = null;
 
     if (platformHelper.isElectron()) {
-
       platform = electronPlatform;
-
     } else if (platformHelper.isAndroid()) {
-
       platform = cordovaPlatform;
-
     }
 
+    return platform;
+  }
+
+  toggleFullScreen() {
```

Comment:
Should all full-screen functionality go to a separate controller/component?

Reply:
It's literally just this one function + the initialization of the short cut (app_controller) and button (startup.js). So it's not all that much ...

Reply:
I moved that functionality to `fullscreen_controller.js` now!